test/system_SUITE.erl

```diff
@@ -134,6 +149,17 @@ generate_expired_token(Config, Scopes) ->
           end,
     ?UTIL_MOD:sign_token_hs(?UTIL_MOD:expired_token_with_scopes(Scopes), Jwk).
 
+generate_expirable_token(Config, Seconds) ->
+    generate_expirable_token(Config, ?UTIL_MOD:full_permission_scopes(), Seconds).
+
+generate_expirable_token(Config, Scopes, Seconds) ->
+    Jwk = case rabbit_ct_helpers:get_config(Config, fixture_jwk) of
+              undefined -> ?UTIL_MOD:fixture_jwk();
+              Value     -> Value
+          end,
+    Expiration = os:system_time(seconds) + timer:seconds(Seconds),
```

Comment:
This should be `Expiration = os:system_time(seconds) + timer:seconds(Seconds),` otherwise the number of milliseconds in `Seconds` is added to now, which makes the token expires much later than expected.